config/default.yml

```diff
@@ -3928,7 +3928,7 @@ Style/SafeNavigation:
                   safe navigation (`&.`).
   Enabled: true
   VersionAdded: '0.43'
-  VersionChanged: '0.44'
+  VersionChanged: '0.56'
```

Comment:
They are duplicated since `VersionChanged` was added.
https://github.com/rubocop-hq/rubocop/blob/b0e724d3f23a1489ed83cea52fd02830165f88b9/config/default.yml#L3873-L3890

v0.56.0 has a changelog entry for this cop, so I choose `0.56`.
https://github.com/rubocop-hq/rubocop/blob/dfbb6fd1d7d25342f942dfe43bd4f1d6a603d9bf/CHANGELOG.md#changes-11